tests/__tests__/sitka.test.ts

```diff
@@ -1,17 +1,97 @@
+import { Dispatch } from "redux"
+import { createSitkaAndStore } from "../sitka-test"
+import { createAppStore, Sitka } from "../../src/sitka"
+
+export class SitkaMock<T = {}> extends Sitka {
+  public registeredModules: T
+ }
+
 describe("Sitka", () => {
 
-  test(`setDispatch dispatch was called`, () => {
+  describe('setDispatch', () => {
+    // Setup
+    const { sitka, store } = createSitkaAndStore()
```

Comment:
create new sitka and store per test, don't reuse the same one, since those are stateful, and in the future intended interactions between tests might happen